apps/omg_child_chain/test/omg_child_chain/block_queue/core_test.exs

```diff
@@ -19,6 +19,7 @@ defmodule OMG.ChildChain.BlockQueue.CoreTest do
   import ExUnit.CaptureLog
```

Comment:
I think a pattern that is quite good and I usually follow is to name the argument variables I'm using:
```
 Core.child_block_nums_to_init_with(0, 3000, @child_block_interval, 2)
```
I would call this:
```
mined_num = 0
until_child_block_num = 300, 
finality_threshold = 2
 Core.child_block_nums_to_init_with(mined_num, until_child_block_num, @child_block_interval, finality_threshold)
```
using this pattern, I do not need to store in memory what these arguments are or what they mean. I think we should follow this...